packages/engine/src/interaction/systems/GrabbableSystem.tsx

```diff
@@ -377,7 +373,18 @@ const execute = () => {
    */
   if (isClient)
     for (const entity of grabbableQuery.enter()) {
```

Comment:
lets move this enter query into a reactor mount

Reply:
moved this into the component instead on mount, did the same for the unmount logic